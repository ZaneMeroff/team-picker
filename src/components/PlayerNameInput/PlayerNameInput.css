.player-name-input {
  height: 40px;
  width: 220px;
  margin: 6px;
  color: #000;
  font-size: 100%;
  border-radius: 10px;
  border: 2px solid #62af82;
  text-align: center;
}

.player-name-inputs-container {
  height: 500px;
  width: 300px;
  display: flex;
  flex-direction: column;
  align-items: center;
  overflow-y: scroll;
  margin-top: 10px;
}

.player-name-input-div {
  height: 45px;
  margin: 3px;
}
