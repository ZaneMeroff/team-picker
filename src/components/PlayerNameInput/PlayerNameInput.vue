<template>
  <div class='player-name-input-div'>
    <input
      placeholder='player name...'
      class='player-name-input'
      v-model='name'
      @change="updateNames(name, id)"/>
  </div>
</template>

<script>
  export default {
    name: 'PlayerNameInput',
    props: {
      updateNames: Function,
      id: Number
    },
    data() {
      return {
        name
      }
    },
    methods: {}
  }
</script>

<style>
  @import './PlayerNameInput.css';
</style>
