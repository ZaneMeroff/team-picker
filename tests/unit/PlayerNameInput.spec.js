import { shallowMount } from '@vue/test-utils'
import PlayerNameInput from '../../src/components/PlayerNameInput/PlayerNameInput.vue'

describe('PlayerNameInput.vue', () => {

  it('component renders and matches snapshot', () => {
    const wrapper = shallowMount(PlayerNameInput)
    expect(wrapper).toMatchSnapshot()
  })

})
