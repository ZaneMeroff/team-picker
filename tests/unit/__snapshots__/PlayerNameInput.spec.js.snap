// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerNameInput.vue component renders and matches snapshot 1`] = `<div class="player-name-input-div"><input placeholder="player name..." class="player-name-input"></div>`;
